### DISCUSSION TITLE
IG doesn't come and open image in foreground even when it should?

### DISCUSSION THREAD

**Discussion by mistergom** [d2phap/ImageGlass#1328] [General]
Hello!
This question is following the last posts of this discussion:
https://github.com/d2phap/ImageGlass/issues/620

I read that:
_With "allow multiple instances" OFF then yes, IG doesn't currently force the single window instance to the front. The exception to this is when the IG instance is minimized: the window pops front when IG restores from minimized._

Why?
I'm graphic designer, juggling between a lot of windows when I work, and I would need that IG come in foreground when I double-click on an image, wherever it is... Because I need to see it immediately...
When several windows are one above the other, IG stays in background and/or in place...

Any thoughts?
Thanks!

**Comment by d2phap** [comment]
Replied in https://github.com/d2phap/ImageGlass/issues/620#issuecomment-1111121369
